MinecraftClient/Resources/lang/de.ini

```diff
@@ -0,0 +1,474 @@
+mcc.forge=Prüfe ob der Server Forge benutzt...
+mcc.forgeforce=Zwang-Aktivierung des Forge supports.
+mcc.resolve=Löse auf {0}...
+mcc.found=§8Server gefunden {0}:{1} für Adresse {2}
+mcc.not_found=§8SRV lookup fehlgeschlagen für {0}\n{1}: {2}
+mcc.retrieve=Erhalte Server Info...
+mcc.restart=Starte Minecraft Console Client neu...
+mcc.restart_delay=Warte {0} Sekunden vor Neustart...
+mcc.server_version=Server Version : 
+mcc.disconnected=Mit keinem Server verbunden. Schreibe '{0}help' für weitere Hilfe.
+mcc.press_exit=Oder drücke Enter um den Minecraft Console Client zu verlassen.
+mcc.version_supported=Version wird unterstützt.\nMelde an...
+mcc.single_cmd=§7Befehl §8 {0} §7 gesendet.
+mcc.joined=Server erfolgreich beigetreten.\nTippe '{0}quit', um ihn wieder zu verlassen.
+mcc.reconnect=Warte 5 Sekunden ({0} Versuche übrig)...
+mcc.disconnect.lost=Verbindung wurde verloren.
+mcc.disconnect.server=Server hat die Verbindung abgebrochen :
+mcc.disconnect.login=Login fehlgeschlagen :
+mcc.link=Link: {0}
+mcc.player_dead_respawn=Du bist gestorben. Automatischer Respawn nach 1 Sekunde.
+mcc.player_dead=Du bist gestorben. Schreibe /respawn zum Respawnen.
+mcc.server_offline=§8Der Server läuft im offline Modus.
+mcc.session=Überprüfe Session...
+mcc.session_fail=Konnte Session nicht überprüfen.
+mcc.server_protocol=§8Server Version : {0} (protocol v{1})
+mcc.with_forge=, mit Forge
+mcc.handshake=§8Handshake erfolgreich. (Server ID: {0})
+
+
+[debug]
+# Messages from MCC Debug Mode
+debug.color_test=Farbtest: Dein Terminal sollte {0} anzeigen.
+debug.session_cache_ok=§8Sitzungsdaten wurden erfolgreich vom Speicher geladen.
+debug.session_cache_fail=§8Es konnten keine Sitzungsdaten vom Speicher geladen werden.
+debug.session_id=Erfolgreich. (session ID: {0})
+debug.crypto=§8Crypto keys & hash wurden generiert.
+debug.request=§8Starte Anfrage zu {0}
+
+[error]
+# Error messages from MCC
+error.ping=Ping an IP fehlgeschlagen.
+error.unsupported=Kann nicht zum Server verbinden : Diese Version wird nicht unterstützt !
+error.determine=Konnte Server Version nicht bestimmen.
+error.forgeforce=Der zwanghafte Forge Support wird für diese Minecraft Version nicht unterstützt!
+error.login=Minecraft Login fehlgeschlagen : 
+error.login.migrated=Account wurde migriert, benutze eine E-Mail als Nutzername.
+error.login.server=Login Server nicht erreichbar. Bitte versuche es später erneut.
+error.login.blocked=Falsches Passwort, gesperrte IP oder zu viele Login Versuche.
+error.login.response=Ungültige Antwort des Servers.
+error.login.premium=Dies ist kein premium Benutzer.
+error.login.network=Netzwerkfehler.
+error.login.ssl=SSL Fehler.
+error.login.unknown=Unbekannter Fehler.
+error.login.ssl_help=§8Anscheinend hast du dieses Programm mit MONO gestartet.\nBeim ersten Mal, musst du HTTPS Zertifikate mit folgendem Befehl importieren:\nmozroots --import --ask-remove
```

Comment:
```suggestion
error.login.ssl_help=§8Anscheinend hast du dieses Programm mit MONO gestartet.\nBeim ersten Mal musst du HTTPS-Zertifikate mit folgendem Befehl importieren:\nmozroots --import --ask-remove
```

Reply:
https://www.websecurity.digicert.com/de/de/security-topics/what-is-ssl-tls-https